TEMPLATE.md

```diff
@@ -47,3 +47,23 @@ Below are the repository settings and branch protection rules which you will nee
 ## Require status check to pass before merging
 
 `TODO add image` - search for "Add me as branch protection" and just use that one.
+
+<br />
+
+## Automatic workflows triggered on pushing code to different branches
+
```

Comment:
Maybe easier to use the names of files..?

![image](https://github.com/Aibel365/python-app-template/assets/94840334/8991353c-cb1f-4c70-9527-c8e69ffd8ce6)

Reply:
auto_pull_request_review.yaml

* on pull_request_target
* reviews code based on configuration set in the action_config.toml, and reports back to pull request with status

auto_pull_request_merge.yaml

* on pull_request (if merged)
* generates new release/changelog -> this will again trigger tag push...

auto_tag_push.yaml

* on tag push 
* Triggers action to build docker/gitops if enabled in action_config.toml


auto_push_generate_pre_release.yaml

* on push (as long as its not main branch)
* Triggers build of "next image" and update gitops/docker if enabled in action_config.toml